server.js

```diff
@@ -1,236 +1,10 @@
-const _ = require('lodash');
-const jwt = require('jwt-simple');
-const expressStaticGzip = require('express-static-gzip');
-const express = require('express');
-const app = express();
+require('app-module-path').addPath(__dirname);
 
-const homebrewApi = require('./server/homebrew.api.js');
-const GoogleActions = require('./server/googleActions.js');
-
-// Serve brotli-compressed static files if available
-app.use('/', expressStaticGzip(`${__dirname}/build`, {
-	enableBrotli    : true,
-	orderPreference : ['br'],
-	index           : false
-}));
-
-process.chdir(__dirname);
-
-//app.use(express.static(`${__dirname}/build`));
-app.use(require('body-parser').json({ limit: '25mb' }));
-app.use(require('cookie-parser')());
-app.use(require('./server/forcessl.mw.js'));
-
-const config = require('nconf')
-	.argv()
-	.env({ lowerCase: true })
-	.file('environment', { file: `config/${process.env.NODE_ENV}.json` })
-	.file('defaults', { file: 'config/default.json' });
```

Comment:
`config` object is actually being used in `server.js`, so this code doesn't work as-is

Reply:
Refactored code a little bit in bfdde9104369dd0fc652a79ac6815a8ca60c5450 to fix this issue